build.gradle

```diff
@@ -44,6 +44,14 @@ plugins {
   id "io.swagger.core.v3.swagger-gradle-plugin" version "2.2.0"
 }
 
+spotlessCheck.enabled = false
```

Comment:
Could you add comments to explain why we have to disable `spolessCheck`

Reply:
To be honest: this line is not necessary (given a fact that spotless gradle plugin 6.13.0 tasks can be run in isolation, but not combined with other tasks: https://github.com/diffplug/spotless/issues/1572)

Reply:
Line removed, PR updated/force-pushed. 

Thanx for a review @chia7712 !